__tests__/Unit/Components/Header/Header.test.tsx

```diff
@@ -0,0 +1,224 @@
+import { Header } from '@/components/Header';
+import { render, screen } from '@testing-library/react';
+
+/* eslint-disable */
+const useRouter = jest.spyOn(require('next/router'), 'useRouter');
+
+describe('Header without dev mode', () => {
+    it('should have Tasks category', () => {
+        useRouter.mockImplementation(() => {
+            return {
+                pathname: '/',
+                query: {},
+            };
+        });
+        render(<Header />);
+        const element = screen.queryByText('Tasks');
+        expect(element).toBeInTheDocument();
+        expect(element?.classList.contains('active')).toBeTruthy();
+    });
+
+    it('should have Issues category', () => {
+        useRouter.mockImplementation(() => {
+            return {
+                pathname: '/issues',
+                query: {},
+            };
+        });
+        render(<Header />);
+        const element = screen.queryByText('Issues');
```

Comment:
issues task is available by default in the header component I guess, so why are we testing it after changing the route, this test of queryByText of issues will pass even if it is written in line 16 I guess

Reply:
These tests are to make sure that everything appears as it should be in both normal and dev mode.
Let's say some developer makes changes and by mistake Issues tab is not coming up anymore. Then it will fail and the developer will understand there is some mistake.

Reply:
fixed.